src/sqerl_pgsql_client.erl

```diff
@@ -178,23 +190,23 @@ init(Config) ->
             false -> undefined
         end,
     case pgsql:connect(Host, User, Pass, Opts) of
-        {error, timeout} ->
-            {stop, timeout};
```

Comment:
What happens on connect timeouts now then?  By "no longer handles connect timeout" do you mean that it no longer returns '{error, timeout}' but just returns the standard error record that we handle below, or does something else happen?